build.gradle.kts

```diff
@@ -38,21 +38,17 @@ val versionShadowPlugin = "7.1.2"
 val versionSmallryeOpenApi = "2.1.23"
 val versionSpotlessPlugin = dependencyVersion("versionSpotlessPlugin")
 
-rootProject.extra["versionAsm"] = versionAsm
-
-rootProject.extra["versionErrorPronePlugin"] = versionErrorPronePlugin
-
-rootProject.extra["versionJandex"] = versionJandex
-
-rootProject.extra["versionJandexPlugin"] = versionJandexPlugin
-
-rootProject.extra["versionProtobufPlugin"] = versionProtobufPlugin
-
-rootProject.extra["versionQuarkus"] = versionQuarkus
-
-rootProject.extra["versionShadowPlugin"] = versionShadowPlugin
-
-rootProject.extra["versionSmallryeOpenApi"] = versionSmallryeOpenApi
+mapOf(
+    "versionAsm" to versionAsm,
+    "versionErrorPronePlugin" to versionErrorPronePlugin,
+    "versionJandex" to versionJandex,
+    "versionJandexPlugin" to versionJandexPlugin,
+    "versionProtobufPlugin" to versionProtobufPlugin,
+    "versionQuarkus" to versionQuarkus,
+    "versionShadowPlugin" to versionShadowPlugin,
+    "versionSmallryeOpenApi" to versionSmallryeOpenApi
```

Comment:
nit: trailing comma in these collection literals maybe

Reply:
that's a syntax error